src/app/query/bitcoin/contract/send-accepted-bitcoin-contract-offer.ts

```diff
@@ -2,6 +2,8 @@ export async function sendAcceptedBitcoinContractOfferToProtocolWallet(
   acceptedBitcoinContractOffer: string,
   counterpartyWalletURL: string
 ) {
+  // TODO: refactor to use `axios`
```

Comment:
Usually I'd make an issue and then add the issue in the comment here 
`#1234 TODO: .... `

Reply:
done 👍🏼